### PR TITLE
Fixing issue #279: removing arguments.callee, which is deprecated.

### PR DESCRIPTION
**Pull request by robinfhu**
Also adding some better comments explaining nv.addGraph

### CODE REVIEW

**Comment by robinfhu** [comment]
@liquidpele please review.

File: src/core.js [hunk 1]
```diff
@@ -97,18 +98,38 @@ nv.render = function render(step) {
 
         nv.render.queue.splice(0, i);
 
-        if (nv.render.queue.length) setTimeout(arguments.callee, 0);
+        if (nv.render.queue.length) {
+            setTimeout(renderLoop, 0);
```
Comment (joshjordan) [inline src/core.js]: We can omit the second param when you want `0`

File: src/core.js [hunk 2]
```diff
@@ -97,18 +98,38 @@ nv.render = function render(step) {
...
+        }
         else {
             nv.dispatch.render_end();
             nv.render.active = false;
         }
-    }, 0);
+    };
+
+    setTimeout(renderLoop,0);
```
Comment (joshjordan) [inline src/core.js]: Ditto.

File: src/core.js [hunk 3]
```diff
@@ -97,18 +98,38 @@ nv.render = function render(step) {
...
+or 
+
+nv.addGraph(<generate Function>, <callback Function>)
+
+The generate function should contain code that creates the NVD3 model, sets options
+on it, adds data to an SVG element, and invokes the chart model. The generate function
+should return the chart model.  See examples/lineChart.html for a usage example.
+
+The callback function is optional, and it is called when the generate function completes.
+*/
```
Comment (joshjordan) [inline src/core.js]: This is great work, but let's try to keep completely separate pieces of work in separate commits.

Comment (robinfhu) [inline src/core.js]: Yep will keep in mind
